src/test/java/funda/prog101/HeaterTest.java

```diff
@@ -10,22 +10,21 @@
 	@Test
 	public void SwichHeater_TurnOn_True() {
```

Comment:
you can use camel case naming convention  e.g. **t**his**I**s**N**ame**E**xample()